e2e/provision/install_sandbox.sh

```diff
@@ -75,7 +75,7 @@ fact_caching_connection = /tmp
 EOT
 
 # Management cluster creation
-ansible_cmd="$(command -v ansible-playbook) -i 127.0.0.1, playbooks/cluster.yml --tags ${ANSIBLE_TAG} "
+ansible_cmd="$(command -v ansible-playbook) -i 127.0.0.1, playbooks/cluster.yml --tags ${ANSIBLE_TAG:-all} "
```

Comment:
@radoslawc FYI I believe that it is taken care at https://github.com/nephio-project/test-infra/blob/main/e2e/provision/init.sh#L84-L88 .. is that insufficient?

Reply:
@vjayaramrh yes, that's correct, but it has no way of knowing that it's being sourced by init.sh where this variable gets populated and when the script has set -o nounset it fails on unbound variable which we see in logs. It exits with status 1 failing whole job.